subgraphs/saucerswap/v1/README.md

```diff
@@ -55,3 +53,57 @@ This query fetches aggredated data from all uniswap pairs and tokens, to give a
   }
 }
 ```
+
+### Subgraph Endpoint 
+
+Synced at for mainnet: https://mainnet-thegraph.swirldslabs.com/subgraphs/name/saucerswap/saucerswap-v1?selected=playground
+
+Synced at for testnet: https://testnet-thegraph.swirldslabs.com/subgraphs/name/saucerswap/saucerswap-v1?selected=playground
+
+### Transitioning to Saucerswap V1: Key Adjustments
+
+The Saucerswap V1 subgraph is originally a fork of Uniswap v2 subgraph, but with some modifications to support Saucerswap V1.
+- All references to ETH and WETH have been replaced with HBAR and WHBAR respectively.
+- Token static definitions have been replaced with popular tokens on Hedera and Saucerswap.
+- Factory contract address has been updated to Saucerswap V1 factory contract address.
+- Abis have been updated to Saucerswap V1 abis, since the Saucerswap V1 contracts have different function signatures in some cases. Smart contracts to generate ABI files can be found in their respective repositories.
+    - Factory and Pool: https://github.com/saucerswaplabs/saucerswaplabs-core
+    - Periphery: https://github.com/saucerswaplabs/saucerswap-periphery
+
+## Setup
+
+### Pre-requisites
+1. yarn
+2. thegraph cli
+3. nodejs
+4. HederaTheGraph Index Server
+
+### Getting Started
+
+The subgraph is able to run on 2 different networks: `mainnet` and `testnet`, for each network there is an example `.env` file, `.env.mainnet` and `.env.testnet` respectively.
```

Comment:
I forget why doesn't it work on previewnet. Just due to the cadence of resets or is there some technical reasons?

Reply:
Saucerswap team does not maintain a `previewnet` environment, I guess is not worth the set-up for the time it lives, so they only have testnet and mainnet...